config/base.config.json.example

```diff
@@ -12,6 +12,7 @@
   },
   "debug": false,
   "systemImagesPath": "path/to/system/images/",
+  "onMigrationExecCmd": "drbl-ocs -b -g auto -x -h \"#macaddress#\" -p reboot startdisk restore \"#imagepath#\" sda",
```

Comment:
In `base.config.json` and `INSTALLATION.md` the command is named `onMigrationExecCmd`. By default `onMigrationExecCmd` contains a `drbl-ocs` command. It is presented as a generic command to execute after a migration, in the install description

In `migration.js` the function which launches `onMigrationExecCmd`  is named `execDrblCommand`. Here a very precise function calls a generic one, without explicitly knowing if `onMigrationExecCmd` contains a `drbl` command

I think there is a consistency problem about what are these functions and commands